deepchem/models/tests/test_losses.py

```diff
@@ -198,6 +198,12 @@ def test_sparse_softmax_cross_entropy_pytorch(self):
     expected = [-np.log(softmax[0, 1]), -np.log(softmax[1, 0])]
     assert np.allclose(expected, result)
 
+    labels = torch.tensor([[1], [0]])
```

Comment:
Can you add the same test for TensorFlow?

Reply:
Done.